pkg/scanner/scanner_test.go

```diff
@@ -44,20 +44,27 @@ func TestScan(t *testing.T) {
 		},
 		{
 			Metadata:   files[2],
+			Lines:      20,
+			CodeLines:  15,
+			BlankLines: 4,
+			Comments:   4,
```

Comment:
The expected values are incorrect, please run the test and verify if the suggested value is correct

```suggestion
			Lines:      24,
			CodeLines:  15,
			BlankLines: 5,
			Comments:   4,
```

Reply:
Ah good catch. Had missed it due to rebasing